documentation/deploy-page.md

```diff
@@ -0,0 +1,10 @@
+# Deploy page
+
+Right now the offline page is a sign that is enabled via toggle, and with mvc offline.
+To see how to enable and disable the deploy page check out this [example](https://www.screencast.com/t/FizkXB8kt0iJ)
```

Comment:
What do you think about write and add examples? in my opinion, having a json here could help to understand and to represent the behavior. The video should be a plus, not the main documentation source.

Reply:
Ok, I didn´t know how much to add here.

Reply:
Adding toggle location here would be bad right? Since this is very public.

Reply:
Mmm, it is public anyway, you can see it in the code.